velox/functions/prestosql/Arithmetic.h

```diff
@@ -124,7 +140,7 @@ struct DivideFunction {
 };
 
 template <typename T>
-struct IntervalDivideFunction {
+struct IntervalDayTimeDivideFunction {
```

Comment:
Is it possible to extract a common re-usable templatized function for IntervalDayTime and IntervalYearMonth as they seem to have the same logical structure ?